homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -296,25 +307,36 @@
     ATTR_MODE: "mode",
     ATTR_BUZZER: "buzzer",
     ATTR_CHILD_LOCK: "child_lock",
-    ATTR_TARGET_HUMIDITY: "target_humidity",
     ATTR_LED_BRIGHTNESS: "led_brightness",
-    ATTR_USE_TIME: "use_time",
-    ATTR_HARDWARE_VERSION: "hardware_version",
 }
 
 AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER = {
     **AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_COMMON,
+    ATTR_TARGET_HUMIDITY: "target_humidity",
+    ATTR_HARDWARE_VERSION: "hardware_version",
+    ATTR_USE_TIME: "use_time",
     ATTR_TRANS_LEVEL: "trans_level",
     ATTR_BUTTON_PRESSED: "button_pressed",
 }
 
 AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_CA_AND_CB = {
     **AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_COMMON,
+    ATTR_TARGET_HUMIDITY: "target_humidity",
+    ATTR_HARDWARE_VERSION: "hardware_version",
```

Comment:
Static info is not meant for state attributes unless it's essential for automations.

Reply:
These 2 attributes were moved from `AVAILABLE_ATTRIBUTES_AIRHUMIDIFIER_COMMON` to more specific implementations because they are not common for all humidifiers and not supported by `shuii.humidifier.jsq001`.